### PR TITLE
Add logging mechanisms to init scripts

### PR DESCRIPTION
**Pull request by kaggis**
+ Add logging options to global cfg 
+ Add logging support to upload metric script. 

[To be done] Must add the same logging support to the remaining scripts too

### CODE REVIEW

**Comment by kaggis** [comment]
Please review 

/cc @pkoro

File: standalone/job_status_detail.py [hunk 1]
```diff
@@ -1,136 +1,161 @@
...
+    ArConfig.read(fn_ar_cfg)
+
+    # Initialize logging
+    log_mode = ArConfig.get('logging', 'log_mode')
+    log_file = ArConfig.get('logging', 'log_file')
+    log_level = ArConfig.get('logging', 'log_level')
+    logger = init_logger(log_mode, log_file, log_level, '[job_cycle.py]')
+
+    # Initialize logging
+    log_mode = ArConfig.get('logging', 'log_mode')
```
Comment (pkoro) [inline standalone/job_status_detail.py]: lines 41-43 seem duplicates of lines 35-37. Maybe it's ok but please check if the second part is not needed.

File: standalone/job_status_detail.py [hunk 2]
```diff
@@ -1,136 +1,161 @@
...
+    log_file = ArConfig.get('logging', 'log_file')
+    log_level = ArConfig.get('logging', 'log_level')
+    logger = init_logger(log_mode, log_file, log_level, 'job_status.py]')
```
Comment (pkoro) [inline standalone/job_status_detail.py]: Minor typo in process name. please check

**Comment by skanct** [comment]
Couple of questions/remarks:

1. Why in the configuration file you set log_file when log_mode is syslog?
2. Please add documentation (how the logging is supposed to be configured?)

**Comment by kaggis** [comment]
- When log_mode = syslog log_file is ignored. We can have log_file=none in the config file or you want to have the ability to have it absent as a parameter?

File: standalone/job_status_detail.py (hunk 2 above)
Comment (kaggis) [inline standalone/job_status_detail.py]: Indeed typo - fixed

File: standalone/job_status_detail.py (hunk 1 above)
Comment (kaggis) [inline standalone/job_status_detail.py]: fixed

**Comment by skanct** [comment]
log_file should be ignored either if it is set or if it is absent, when log_mode is syslog. Also how about some unit test regarding this functionality?

**Comment by pkoro** [comment]
+1 (wrt the pull request)

**Comment by kaggis** [comment]
This pull request has conflicts with the previous merges! Must resolve them

**Comment by kaggis** [comment]
Too many conflicts i will delete the origin branch and rework it

**Pull request closed without merging**